### PR TITLE
Update `YOLO` description to be more specific

### PR DESCRIPTION
**Pull request by sgrokym**

### CODE REVIEW

**Comment by Schweinepriester** [comment]
Thank you for the PR, but have you done any testing around this?
Because at least in the past this wasn't necessary, since for example my unlock doesn't fulfill said criteria: https://github.com/Schweinepriester?achievement=yolo&tab=achievements

**Comment by Schweinepriester** [comment]
> "merge without a review, but with a pending review request"

exactly this is my current status (proposed [here](https://github.com/Schweinepriester/github-profile-achievements/issues/15#issuecomment-1153285742) and then verified with [the PR](https://github.com/Schweinepriester/achievement-testarea/pull/4) which unlocked it for me).

but we know for sure they tinkered after the release, so…

**Comment by sgrokym** [comment]
OK, let me change the sentence a bit.
How about this one.
`Merged a reviewable pull request without code review`

In my experience the following did not work.
https://github.com/sgrokym/adr-jan/pull/1
So, I think `Merged a reviewable pull request without code review` is more appropriate than `Merged a pull request without code review`

FYI, I don't have YOLO badge so I can test which one does not work.

**Comment by FleetAdmiralJakob** [comment]
Pls. make the YOLO Description more specific because I don't get this achievement

**Comment by Lymengchun** [comment]
good

**Comment by DYL4NonTop** [comment]
good

**Comment by akirataguchi115** [comment]
Fix the the condition to this:

**Merged a pull request with reviewers requested but before any reviews**

**Review by akirataguchi115 — COMMENTED** [review]
Fix the the condition to this:

**Merged a pull request with reviewers requested but before any reviews**

**Comment by ParanoidUser** [comment]
Quick update on the experiment [YOLO PR #1](https://github.com/ParanoidUser/yolo/pull/1). We confirmed that the guide [how to earn YOLO badge](https://github.com/Schweinepriester/github-profile-achievements/discussions/6) is working.  

As described there all you need is ~~love~~ follow the steps:
 1. Create any PR on any repository
 2. Assign another person as a reviewer. If you don't have any, invite somebody to collaborate.
 3. Accept the warning and merge the PR without review
 4. Celebrate your new profile badge.

Let me know if you have any questions.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Starstruck | ![Starstruck Badge](/images/starstruck-default.png) | ✔️ | Crea
 ||| <!-- this empty row is intentional to separate -->
 Galaxy Brain | ![Galaxy Brain Badge](/images/galaxy-brain-default.png) | ✔️ | 2 accepted answers
 Pull Shark | ![Pull Shark Badge](/images/pull-shark-default.png) | ✔️ | 2 pull requests merged
-YOLO | ![YOLO Badge](/images/yolo-default.png) | ✔️ | Merged a pull request without code review
+YOLO | ![YOLO Badge](/images/yolo-default.png) | ✔️ | Merged a pull request on repository with multiple contributors without code review
```
Comment (port19x) [inline README.md:25]: as suggested by akira
```suggestion
YOLO | ![YOLO Badge](/images/yolo-default.png) | ✔️ | Merged a pull request with reviewers requested but before any reviews
```

**Comment by marcos-jesus** [comment]
resolve conflict ?

**Comment by marcos-jesus** [comment]
@ParanoidUser It's good!

File: README.md (hunk 1 above)
Comment (faizqadri234) [reply README.md:25]: I would suggest 
`Merged a pull request with at least one pending reviewer`

**Review by Abdullah-As-Sadeed — COMMENTED** [review]
Yes, it'll be more helpful.

**Pull request closed without merging**